### ISSUE TITLE
Accessibility Improvement: missing lang attribute on html tag 

### ISSUE THREAD

**Issue by JAGrenier** [wnbrb/wnb-rb-site#39]
currently missing a language attribute on html tag. 
This attribute communicates to screen readers the language content the page is written in.

**Comment by JAGrenier** [comment]
I would be happy to put in a PR to address this!

**Comment by egiurleo** [comment]
Thanks, @JAGrenier! Looking forward to your PR. :D

**Comment by egiurleo** [comment]
Closed with #42.
